package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@oat-sa-private/e2e-runner",
+  "version": "0.1.0",
+  "displayName": "OAT end-to-end test runner",
+  "description": "End-to-end test runner for OAT applications, based on Cypress",
+  "publishConfig": {
+    "access": "restricted"
+  },
+  "license": "GPL-2.0",
+  "scripts": {
+    "lint": "eslint plugins support"
+  },
+  "keywords": [
+    "oat",
+    "e2e",
+    "testrunner"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oat-sa/e2e-runner.git"
+  },
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "bugs": {
+    "url": "https://github.com/oat-sa/e2e-runner/issues"
+  },
+  "homepage": "https://github.com/oat-sa/e2e-runner#readme",
+  "devDependencies": {
+    "@oat-sa/eslint-config-tao": "^1.0.0",
+    "@oat-sa/prettier-config": "^0.1.1",
+    "eslint": "^7.20.0",
+    "eslint-plugin-es": "^4.1.0",
+    "eslint-plugin-jsdoc": "^32.2.0",
```

Comment:
with the last version of `@oat-sa/eslint-config-tao` you don't need to handle plugins yourself